.github/workflows/main.yaml

```diff
@@ -0,0 +1,88 @@
+name: AndcultureCode.JavaScript.Testing
+
+on:
+    push:
+        branches: ["*"]
+    pull_request_target:
+        branches: [main]
+    workflow_dispatch:
+
+env:
+    CODECOV_TOKEN: ${{secrets.CODECOV_TOKEN}}
+    SLACK_BOT_TOKEN: ${{secrets.SLACK_BOT_TOKEN}}
+
+jobs:
+    notify_start:
+        runs-on: ubuntu-latest
+        # Ideally this condition would be promoted to a workflow-level environment variable and shared
+        # but this doesn't seem to be supported yet https://github.community/t/how-to-set-and-access-a-workflow-variable/17335/6
+        if: github.repository_owner == 'AndcultureCode'
+        steps:
+            - uses: voxmedia/github-action-slack-notify-build@v1
+              with:
+                  channel: company-andculture-code-ci
+                  color: "#3278BD"
+                  status: STARTED
+    build:
+        runs-on: ${{matrix.os}}
+        needs: [notify_start]
+        if: always()
+        strategy:
+            matrix:
+                include:
+                    - {
+                          os: windows-latest,
+                          node: 8.16.2,
+                          allowed-to-fail: false,
+                      }
+                    - {
+                          os: windows-latest,
+                          node: 14.15.5,
+                          allowed-to-fail: true,
+                      }
+        continue-on-error: ${{matrix.allowed-to-fail}}
+
+        steps:
+            - uses: actions/checkout@v2
+
+            - name: Setup node ${{matrix.node}}
+              uses: actions/setup-node@v2
+              with:
+                  node-version: ${{matrix.node}}
+
+            - name: npm install
+              run: |
```

Comment:
The pipe is for multi-line commands. I think you could do something like this if you wanted to chain them together
```yaml
- name: build
- run: |
         npm install
         npm run build
         npm run coverage
```

Reply:
True. I think an earlier version of the `and-cli` script had multiple commands in one `run` step (there might still be some) - I prefer having them split up and named so they are discrete steps in the UI, but I'm not too set on it either way